lib/jsonld_serializers.js

```diff
@@ -257,12 +257,13 @@ class ResourceSerializer extends JsonLdItemSerializer {
     // Add hasVol and hasDate properties
     stmts.hasItemVolumes = false
     stmts.hasItemDates = false
+    const numItems = this.body.numItems[0] || this.body.numItemsTotal[0]
```

Comment:
Won't this be a property access error if `numItems` is `undefined`?

Reply:
fixed